File: src/main/java/com/back/nbe9112team06/domain/timeblock/repository/AvailableTimeRepository.java
package com.back.nbe9112team06.domain.timeblock.repository;

import com.back.nbe9112team06.domain.timeblock.entity.AvailableTime;
import org.springframework.data.jpa.repository.JpaRepository;

public interface AvailableTimeRepository extends JpaRepository<AvailableTime, Integer> {
}

File: src/main/java/com/back/nbe9112team06/domain/timeblock/repository/AvailableTimeRepository.kt
package com.back.nbe9112team06.domain.timeblock.repository

import com.back.nbe9112team06.domain.timeblock.entity.AvailableTime
import org.springframework.data.jpa.repository.JpaRepository

interface AvailableTimeRepository : JpaRepository<AvailableTime, Int>